src/extension-runners/chromium.js

```diff
@@ -334,6 +345,19 @@ export class ChromiumExtensionRunner {
       const ws = new window.WebSocket(
         "ws://${wssInfo.address}:${wssInfo.port}");
 
+      
+      if (${specialStartingUrls.length} > 0)
+      {   
+        const chromeTabList = "${specialStartingUrls}".trim().split(" ")
```

Comment:
Serialize the list of URLs with `${JSON.stringify(specialStartingUrls)}` instead of splitting by strings. Otherwise, if the URL contains a `"`, this could be abused to execute code in the context of the helper extension. That's problematic.